src/main/kotlin/me/partlysanestudios/partlysaneskies/utils/StringUtils.kt

```diff
@@ -316,7 +316,7 @@ object StringUtils {
         }
     }
 
-    fun colorCodeToColor(colorCode: String): Color {
+    fun String.colorCodeToColor(): Color {
         val colorCodeToColor = HashMap<String, Color>()
```

Comment:
I feel like a 

when(this) {
  "§a" -> Color()
  
  is best here

Reply:
what?? creating an entirely new hashmap, assinging a bunch of temporary values, then going through them to find the right one isn't the most efficient???